helm/net-exporter-chart/values.yaml

```diff
@@ -10,6 +10,8 @@ dns:
   port: 1053
   label: coredns
 
+timeout: 5s
```

Comment:
maybe we can set longer timeout by default?

Reply:
what do you have in mind? double? `10s`?